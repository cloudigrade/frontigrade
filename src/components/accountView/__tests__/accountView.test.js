import React from 'react';
import { mount } from 'enzyme';
import AccountView from '../accountView';

describe('AccountView Component', () => {
  it('should render', () => {
    const primaryClick = jest.fn();
    const props = {
      primaryClick
    };

    const component = mount(<AccountView {...props} />);
    expect(component.render()).toMatchSnapshot();

    component.find('button').simulate('click');
    expect(primaryClick).toBeCalled();
  });
});
