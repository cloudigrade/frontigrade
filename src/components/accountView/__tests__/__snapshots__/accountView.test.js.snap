// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountView Component should render 1`] = `
<div
  class="container-fluid"
>
  <div
    class="row"
  >
    <div
      class="blank-slate-pf full-page-blank-slate"
    >
      <div
        class="blank-slate-pf-icon"
      >
        <span
          aria-hidden="true"
          class="pficon pficon-add-circle-o"
        />
      </div>
      <h4
        class="h1 blank-slate-pf-title"
      >
        Lorem Ipsum
      </h4>
      <p
        class="blank-slate-pf-info"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean commodo purus a sem ornare ullamcorper. Duis sed mi mi. Aenean risus ante, hendrerit sed cursus nec, pharetra eu nisi. Nulla in dui id leo dapibus malesuada eu placerat lacus. Phasellus pulvinar magna vel mattis commodo.
      </p>
      <div
        class="blank-slate-pf-main-action"
      >
        <button
          class="btn btn-lg btn-primary"
          type="button"
        >
          Dolor sit amet
        </button>
      </div>
    </div>
  </div>
</div>
`;
